### DISCUSSION TITLE
Aeotec stick Gen5+ to Gen5 network migration

### DISCUSSION THREAD

**Discussion by ivdimitro** [zwave-js/zwave-js#4797] [Q&A] [answered]
Hello, I need to migrate my network from the Aetoec Gen5+ stick to my older Gen5 stick. I was thinking I could just backup/restore from one to another but unfortunately I am getting an error as in the screenshot. 
Basically the SDK versions are incompatible. 

Is there a way of avoiding this version check or the SDK compatibility is absolute must? 
Any other idea of how to migrate the network between the sticks with the least amount of reconnecting devices :)

![Gen5_restore_error](https://user-images.githubusercontent.com/6159024/178710798-d7151d2c-89e0-4947-a5f7-0b088ced34e6.jpg)

**Comment by AlCalzone** [comment, marked as answer]
You must upgrade both sticks to the 1.2 firmware. I think Aeotec pulled the download, but they might give it to you if you ask.

**Reply by ivdimitro** [reply]
Thanks. The update is still available, but It takes some digging to find it. The transfer operation worked. I even made the backup with the windows tool and restored via the web interface. 
Great job on the interoperability.

**Reply by mundschenk-at** [reply]
The update is publicly available again with a disclaimer that you are on your own if it bricks your stick: https://aeotec.freshdesk.com/support/solutions/articles/6000252294-z-stick-gen5-v1-02-firmware-update
